src/widgets/vitals/vital-record.test.tsx

```diff
@@ -3,15 +3,27 @@ import { cleanup, render, wait } from "@testing-library/react";
 import { BrowserRouter, match } from "react-router-dom";
 import VitalRecord from "./vital-record.component";
 import { mockPatient } from "../../../__mocks__/patient.mock";
+import { chartWidgetsConfigMock } from "../../../__mocks__/chart-widgets-config.mock";
 import { useCurrentPatient } from "../../../__mocks__/openmrs-esm-api.mock";
 import { mockVitalSigns, mockVitalData } from "../../../__mocks__/vitals.mock";
 import { performPatientsVitalsSearch } from "./vitals-card.resource";
 import { of } from "rxjs/internal/observable/of";
 import { useRouteMatch } from "react-router";
+import { useConfig, validators } from "@openmrs/esm-module-config";
 
 const mockUseCurrentPatient = useCurrentPatient as jest.Mock;
 const mockUseRouteMatch = useRouteMatch as jest.Mock;
 const mockFetchPatientVitalSigns = performPatientsVitalsSearch as jest.Mock;
+const mockUseConfig = useConfig as jest.Mock;
+const mockIsStringValidatior = validators.isString as jest.Mock;
+
+jest.mock("@openmrs/esm-module-config", () => ({
+  useConfig: jest.fn(),
+  validators: {
+    isString: jest.fn()
+  },
+  validateConfig: jest.fn()
+}));
```

Comment:
You should be able to factor out all these copies of this mock into `__mocks__`

Reply:
If you want :)

Reply:
Gave it a try, wasn't successful. Could you give me pointers or hints?

Reply:
Sure! It should be that all you need to do is more or less copy what's [here](https://github.com/openmrs/openmrs-esm-login/tree/master/__mocks__) and then you wire them up with moduleNameMapper in the [jest config](https://github.com/openmrs/openmrs-esm-login/blob/master/jest.config.json).